src/kili/services/export/format/coco/__init__.py

```diff
@@ -58,10 +58,12 @@ class _CocoFormat(TypedDict):
 
 class CocoExporter(AbstractExporter):
     """
-    Common code for Kili exporters.
+    Common code for COCO exporter.
     """
 
-    download_media = True  # COCO is self-contained and requires paths to the media files
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.download_media = True  # we need the assets for COCO annotation format
```

Comment:
if this differs from the `download_media` arg is false, you can output a warning.